proposals/governance/incubation_dd.md

```diff
@@ -0,0 +1,87 @@
+
+### GitHub
+* 171 official members in the organization
+* About 50 contributors measured by multiple L or greater sized contributions
+* 96 watched tags
+* 2562 stars
+* 615 forks
+
+### Slack
+Slack channels typically get 0-5 new threads per day.
+
+### Downloads
+
+### Integrations
+* Kubernetes / minikube
+* Red Hat OKD / OpenShift 4.x
+* oVirt / Red Hat Virtualization
+
+### Community
+
+#### Communications
+
+Kubevirt uses several technologies to maintain the community.
+
+Source code is stored in git repositories under a GitHub organization provided by the CNCF (https://www.github.com/kubevirt).  GitHub issues are used to track reqs for engineering and bug tracking.  GitHub pull requests are used to peer review contributions.
+
+The project utilizes Google Groups as a mailing list where use, support, proposal topics are discussed.
+
+The project holds a video conference every week via a Zoom account provided by the CNCF.  General topics, support and bug triage are conducted in this meeting.  The project also conducts a weekly meeting based on the topic of performance and scale.  Both meetings are recorded and posted to the project YouTube channel.  Meeting notes are emailed to the general mailing list.
+
+The project uses two channels on the CNCF/Kubernetes Slack server.  #virtualization is used to handle general use and support topics.  #kubevirt-dev is used for developer oriented communication.
+
+KubeVirt utilizes email, website blog and Twitter for important announcements such as version releases.
+
+KubeVirt advertises communications channels via https://kubevirt.io/community as well as project README (https://github.com/kubevirt/kubevirt/blob/master/README.md).
+
+#### End Users
+* NVIDIA ... NVIDIA's latest computing platform is built on open-source projects like Kubernetes and KubeVirt to power products like GeForce NOW with more to come.
+*
+*
+*
+
+### CNCF Sponsored Security Audit
+
+## Features & Roadmap
+
+### Roadmap at Sandbox
+* [DONE] GA v1 API for core KubeVirt APIs
+ * API v1 features need to rely on GA’ed Kubernetes entities, fully fledged (incl e.g. explain, validation)
+ * An OpenAPI definition as the only source of truth for KubeVirt’s API
+ * https://github.com/kubevirt/kubevirt/pull/3349
+* [DONE] Stabilize or replace bridge network binding
+* [DONE] Disk hot add
+* [DONE] IPv6 support
+* [DONE] Device passthrough support
+* [DONE] Memory metrics gathering support
+* [DONE] Affinity / Anti-Affinity rules
+* [DONE] Live-Migration support
+* [DONE] Offline disk snapshots
+* [DONE] SRIOV support
+
+### Future Roadmap
+* [WIP] Non-root VMI Pods
+* [WIP] Persistent containerDisk volumes
+* [WIP] Establish predictable community release and support patterns
+* [WIP] Define a deprecation policy
+* [WIP] Review and Revise User Guide
+* [WIP] Virt-launcher live updates
+* [WIP] Templating mechanism for VMs
+* [WIP] Monitoring and metrics standardization
+* [WIP] CPU pinning support
```

Comment:
We have pinning now. Numa topology support is somethign we're still workign on though that's similar